Src/DependencyCollector/Shared/Implementation/HttpDesktopDiagnosticSourceListener.cs

```diff
@@ -53,7 +53,7 @@ public void OnNext(KeyValuePair<string, object> value)
                     case "System.Net.Http.Request": 
                     {
                         var request = (HttpWebRequest)this.requestFetcherRequestEvent.Fetch(value.Value);
```

Comment:
We need to add DependencyCollectorEventSource.Log.Callback received type of logging here.